doc/getting_started/index.rst

```diff
@@ -51,27 +50,21 @@ First, install the ``west`` binary and bootstrapper:
    # macOS and Windows
    pip3 install west
 
-.. note::
-   See :ref:`gs_python_deps` for additional clarfication on using the
-   ``--user`` switch.
+Some notes on the ``--user`` option:
```

Comment:
My west autocomplete PR (#16013) will force you to rewrite this a bit

Reply:
I know; that's fine. I wanted to get this up before hopping on the plane and plan on rebasing once your PR is in.